src/main/java/mate/academy/exception/DataProcessingException.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.exception;
+
+public class DataProcessingException extends Exception {

```

Comment:
it's better to make exception unchecked, this exception is usually thrown because of dev mistake